packages/protocol/src/certificate/CertificateAuthority.ts

```diff
@@ -39,6 +56,9 @@ export class CertificateAuthority {
     #rootCertBytes?: Bytes;
     #nextCertificateId = BigInt(1);
     #construction: Construction<CertificateAuthority>;
+    #ica!: boolean;
```

Comment:
Why do you needed to enfore existence?  Better make option and  add this.construction.assert() before using it because then better to enforce when using or returning it.